code/game/objects/items/toys.dm

```diff
@@ -511,10 +511,10 @@
 
 /*
  * Syndie stealthy smokebombs!
- */
- /obj/item/toy/snappop/smokebomb
- 	origin_tech = Tc_COMBAT + "=1;" + Tc_SYNDICATE + "=1"
+*/
+/obj/item/toy/snappop/smokebomb
 	flags = FPRINT | NO_THROW_MSG
+	origin_tech = Tc_COMBAT + "=1;" + Tc_SYNDICATE + "=1";
```

Comment:
That semicolon is not needed. The problem on this line was the horrendous indentation.

Reply:
ah ok, i was wondering why it was complaining there